MCE.py

```diff
@@ -1132,7 +1132,7 @@ def mass_scan(f_path) :
 pat_icpu = re.compile(br'\x01\x00{3}.{4}(([\x00-\x22]\x20)|([\x93-\x99]\x19))[\x01-\x31][\x01-\x12].{3}\x00.{4}[\x01\x00]\x00{3}.\x00{3}.{3}\x00.{3}\x00{13}', re.DOTALL)
 
 # AMD - Year 20xx, Month 01-13, LoaderID 00-04, DataSize 00|10|20, InitFlag 00-01, NorthBridgeVEN_ID 0000|1022, SouthBridgeVEN_ID 0000|1022, BiosApiREV_ID 00-01, Reserved 00|AA
-pat_acpu = re.compile(br'\x20[\x01-\x31][\x01-\x13].{4}[\x00-\x04]\x80[\x00\x20\x10][\x00\x01].{4}((\x00{2})|(\x22\x10)).{2}((\x00{2})|(\x22\x10)).{6}[\x00\x01](\x00{3}|\xAA{3})', re.DOTALL)
+pat_acpu = re.compile(br'\x20[\x01-\x31][\x01-\x13].{4}[\x00-\x05]\x80[\x00\x20\x10][\x00\x01].{4}((\x00{2})|(\x22\x10)).{2}((\x00{2})|(\x22\x10)).{6}[\x00\x01](\x00{3}|\xAA{3})', re.DOTALL)

```

Comment:
Oh that very tight range might have made us miss Zen 3 microcodes in the wild already. That's not cool. Maybe I can extend this range a bit for better future proofing. It is currently very strict because AMD's header is very false positive prone in general.

Reply:
That might be a good idea for the future, but for now it should be enough. From what I can see Family 19h has always `05` and Family 17h `04`.